website/static/js/nodeSelectTreebeard.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+var $ = require('jquery');
+var m = require('mithril');
+var ko = require('knockout');
+var Treebeard = require('treebeard');
+var projectSettingsTreebeardBase = require('js/projectSettingsTreebeardBase');
+
+
+function expandOnLoad() {
```

Comment:
The three functions below are copied in 3 different places. See if you can refactor. How about putting these functions in `projectSettingsTreebeardBase`?